test/unit/action/log.spec.js

```diff
@@ -10,6 +10,14 @@ describe('Action Logs Unit Tests', () => {
   let ipc;
   let ipcRenderer;
 
+  const RNFS = {
+    DocumentDirectoryPath: '/foo/bar',
+    readFile: () => Promise.resolve('logs'),
+  };
+  const Share = {
+    share: () => Promise.resolve(true),
+  };
+
```

Comment:
Stubs should be declared via `let` here and initialized using `sinon.stub().resolves(true)`  in the `beforeEach` block below. That way all test case can work with a new stub instance and check how often it was called.